minigalaxy/api.py

```diff
@@ -134,7 +137,7 @@ def get_info(self, game: Game) -> dict:
         return response
 
     # This returns a unique download url and a link to the checksum of the download
-    def get_download_info(self, game: Game, operating_system="linux", dlc_installers="") -> dict:
+    def get_download_info(self, game: Game, operating_system="", dlc_installers="") -> dict:
```

Comment:
Why was this change made?

Reply:
You force "operating_system" = "linux".

In gametile.py, you have "download_info = self.api.get_download_info(self.game, self.api.get_download_version(self.game))"

but if you set "operating_system" = "linux" in apy.py, you always download the linux version of a game even if "self.api.get_download_version(self.game) = windows"

Reply:
Okay, thanks for explaining!